gtk/src/subclass/list_box_row.rs

```diff
@@ -19,6 +19,21 @@ pub trait ListBoxRowImplExt: ObjectSubclass {
     fn list_box_row_activate(&self, list_box_row: &Self::Type);
 }
 
+impl<T: ListBoxRowImpl> ListBoxRowImplExt for T {
+    fn list_box_row_activate(&self, list_box_row: &Self::Type) {
```

Comment:
This should be renamed to `parent_activate` to follow the same naming conventions we use everywhere.

Reply:
Should this be part of this PR? Or a subsequent one?

Reply:
I've done it here.

Reply:
As part of this PR is fine